api/v1alpha8/openstackcluster_types.go

```diff
@@ -120,10 +120,10 @@ type OpenStackClusterSpec struct {
 	// ManagedSecurityGroups determines whether OpenStack security groups for the cluster
 	// will be managed by the OpenStack provider or whether pre-existing security groups will
 	// be specified as part of the configuration.
-	// By default, the managed security groups have rules that allow the Kubelet, etcd, the
-	// Kubernetes API server and the Calico CNI plugin to function correctly.
-	// +optional
-	ManagedSecurityGroups bool `json:"managedSecurityGroups"`
+	// By default, the managed security groups have rules that allow the Kubelet, etcd and the
```

Comment:
I don't see where a comma is missing, I have it between Kubelet and etcd and not after because there is an "and". Please just tell me where it's missing and I'll add a comma.